weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/SchemaMappingStep.tsx

```diff
@@ -0,0 +1,352 @@
+import {Box, Paper, Stack, Tooltip, Typography} from '@mui/material';
+import React, {useEffect, useState} from 'react';
+
+import {parseRef} from '../../../../../react';
+import {Select} from '../../../../Form/Select';
+import {Icon} from '../../../../Icon';
+import {LoadingDots} from '../../../../LoadingDots';
+import {Pill} from '../../../../Tag/Pill';
+import {CopyableId} from '../pages/common/Id';
+import {useWFHooks} from '../pages/wfReactInterface/context';
+import {ObjectVersionSchema} from '../pages/wfReactInterface/wfDataModelHooksInterface';
+import {SmallRef} from '../smallRef/SmallRef';
+import {
+  CallData,
```

Comment:
![Screenshot 2025-02-04 at 18 29 16](https://github.com/user-attachments/assets/890c51f8-fe1f-4c03-ab88-29c29d13e703)

If something is a ref, we are not resolving it... this seems pretty nasty. You could punt on this, but i suspect it will show up for many users

Reply:
I'm updated the PR so that these values are shown via small ref and can't be edited. 

If we want to support editing... do you think it would be useful to resolve the underlying value? It doesn't feel right to break the object lineage, yet I suspect trying to recursively create new versions of objects that are referenced in the dataset will be crazy also.